server.py

```diff
@@ -32,7 +33,30 @@ def get_shopcarts(id):
 
     return jsonify(message), rc
 
-
+######################################################################
+# Create a Shopcart
+######################################################################
+@app.route('/shopcarts/<int:id>', methods=['PUT'])
+def create_shopcart(id):
+    """Creates a shopcart and saves it to database
+        POST Request accepts data as: 
+        - id + prod id (sets default to 1)
+        - id + {prod:quant}
```

Comment:
I think we should also allow just the id which will create a cart with no products.

Reply:
Do we want that? the issue said we would have a product. @thoughts1053 
Given that no shopcart exists
When I POST to /shopcarts/userid
And I make a GET to /shopcarts/userid
Then I should have 1 product in your shopping cart

Reply:
I agree with @carpetri here, I believe we agreed we don't want to create empty shopcarts -
it shouldn't exist until the user wants to add a product. 

If we created empty shopcarts, how would our `prune` Action know which carts were purposely emptied by the user and which carts were just created empty?